src/test/java/net/kaczmarzyk/spring/data/jpa/web/utils/PathVariablesUtil.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2014-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.kaczmarzyk.spring.data.jpa.web.utils;
+
+import net.kaczmarzyk.spring.data.jpa.web.MockWebRequest;
+import org.springframework.web.context.request.RequestAttributes;
+import org.springframework.web.servlet.HandlerMapping;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PathVariablesUtil {
```

Comment:
add javadoc with description of the purpose 

hint: explain why sometimes path varialbes should be set as request attributes and in other cases they don't have to be. You may link to specific section in Spring documentation.